metricbeat/module/munin/node/_meta/data.json

```diff
@@ -0,0 +1,207 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "munin.example",
+        "duration": 115000,
+        "module": "munin"
+    },
+    "metricset": {
+        "name": "node"
+    },
+    "munin": {
+        "example": {
+            "cpu": {
+                "guest": 0,
+                "idle": 194286379,
+                "iowait": 75847,
+                "irq": 0,
+                "nice": 0,
+                "softirq": 64417,
+                "steal": 0,
+                "system": 4743011,
+                "user": 3686223
+            },
+            "df": {
+                "_dev": 0,
+                "_dev_sda1": 17.2858983372179,
+                "_proc_acpi": 0,
+                "_sys_firmware": 0,
+                "_sys_fs_cgroup": 0,
+                "overlay": 17.2858983372179,
+                "shm": 0
+            },
+            "df_inode": {
+                "_dev": 0.00252557938369549,
+                "_dev_sda1": 8.53179355837264,
+                "_proc_acpi": 0.000157848711480968,
+                "_sys_firmware": 0.000157848711480968,
+                "_sys_fs_cgroup": 0.00220988196073355,
+                "overlay": 8.53179355837264,
+                "shm": 0.000157848711480968
+            },
+            "entropy": {
+                "entropy": 3363
+            },
+            "forks": {
+                "forks": 29569079
+            },
+            "fw_conntrack": {
+                "assured": 14,
+                "established": 1,
+                "fin_wait": 0,
+                "nated": 0,
+                "syn_sent": 0,
+                "time_wait": 4,
+                "total": 14,
+                "udp": 0
+            },
+            "fw_forwarded_local": {
+                "forward": 0,
+                "local": 1
+            },
+            "fw_packets": {
+                "forwarded": 0,
+                "received": 823
+            },
+            "if_err_eth0": {
```

Comment:
All this fields seems dynamically generated from the machine network interfaces. I thought this wasn't ideal for a Metricbeat module. Just asking

Reply:
They are names of "plugins", now they are going to go to the `munin.plugin.name` field as values, so we avoid fields explosion. and problems of this kind.

I am open in the future to add some kind of grok-like patterns to extract fields from these names (e.g, extract labels with expressions like `if_err_%{WORD:network_interface}`).